_pages/en/belt-system.md

```diff
@@ -0,0 +1,505 @@
+---
+layout: default
+title: Belt Maturity Assessment System
```

Comment:
I offer y'all: a parrrrrtial [recursive backronym](https://en.wikipedia.org/wiki/Recursive_acronym)! Does this give me government-points 😄 

```suggestion
title: BELT Experience L________ Tracker
```